ols/constants.py

```diff
@@ -7,7 +7,7 @@
 INVALID = "INVALID"
 YAML = "YAML"
 NOYAML = "NOYAML"
-SOME_FAILURE = "some failure"
+REPHRASE = "REPHRASE"
```

Comment:
please prefix these constants to make their usage/intent more clear:

SUBJECT_INVALID
SUBJECT_VALID
CATEGORY_YAML
CATEGORY_NOYAML
CATEGORY_UNKNOWN

Reply:
Thank you Ben @bparees  - Slight change instead of CATEGORY_NOYAML , planning to use CATEGORY_GENERIC